<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HRicette</class>
 <widget class="QWidget" name="HRicette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ricette</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Crea &amp;Nuova Ricetta</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>610</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Duplica &amp;Ricetta</string>
   </property>
  </widget>
  <widget class="QListView" name="lvRicette">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>271</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="cvRigric">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>461</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leID">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teNote">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>461</width>
     <height>64</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>610</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Aggiungi Ingrediente</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>HRicette</receiver>
   <slot>addRecipe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addRecipe()</slot>
 </slots>
</ui>
